packages/api/amplify_api/lib/src/api_plugin_impl.dart

```diff
@@ -240,7 +243,7 @@ class AmplifyAPIDart extends AmplifyAPI {
     return AWSStreamedHttpRequest.delete(
       uri,
       body: body,
-      headers: addContentTypeToHeaders(headers, body),
```

Comment:
The new HTTP client PR moves this logic to AWS common so not needed here (happens without this code).